doc/user/content/overview/_index.md

```diff
@@ -10,185 +10,84 @@ menu:
     weight: 5
 ---
 
-Materialize is a **streaming database** powered by [Timely](https://github.com/TimelyDataflow/timely-dataflow#timely-dataflow) and
+Materialize is a **streaming database** purpose-built for low-latency
+applications. You can use it to solve problems where data must be processed at
+speeds and scales not possible in traditional databases, but without the cost,
+complexity, and development time of most streaming engines.
+
+If you need to reduce the time it takes to refresh queries that run frequently,
+or trigger user-facing actions as soon as events happen, Materialize might be a
+good fit for you! Rather than recalculate results from scratch each time it's
+queried, or serve stale cached results, Materialize continually ingests data
+and keeps results up to date for you as **incrementally updated materialized
+views**.
+
+<br>
+
+| ![streaming_analytics.gif](https://user-images.githubusercontent.com/23521087/233637537-4c2e5133-5a93-4d9d-9800-a0d4ba842c82.gif) |
+|:---------------------------------------------------------------------------------------------------------------------------------:|
+| *As an example, you can use Materialize to keep your dashboards automatically up to date.*                                                           |
+
+## Features
+
+Materialize combines all the desirable features of SQL databases and abstracts
+away all the hard parts required to build a streaming engine horizontally
+scalable, highly available and strongly consistent. To get up and running, all
+you need to know is **SQL**.
+
+### Incrementally updated views
+
+Materialized views are useful to avoid re-running heavy queries that run
+frequently against a database, and typically use caching to serve results
+faster. In traditional databases, materialized views often come with a long
+list of limitations and force you to make a compromise between the freshness of
+the results, the cost of refreshing the view and the complexity of the SQL
+statements you can use.
+
+Materialize supports incrementally updated materialized views that are **always
+fresh**, and allows a much broader set of SQL statements, like views over
+multi-way joins with complex aggregations. How? Under the hood, its engine is
```

Comment:
nit: the multiple `and`s in this sentence feel a little clunky. maybe...

How? Under the hood, Materialize is powered by Timely and Differential Dataflow, systems backed by many years of research and optimized for this exact purpose.